plugins/inputs/mongodb/README.md

```diff
@@ -10,6 +10,7 @@
   ##   mongodb://10.10.3.33:18832,
   ##   10.0.0.1:10000, etc.
   servers = ["127.0.0.1:27017"]
+  gather_db_stats = false
```

Comment:
Guys, we are trying to use gather_per_db stats in InfluxDB 1.0-beta3 and it keeps giving us this error:
Error parsing /apps/dftmongo/bin/telegraf/conf/telegraf.conf, line 50: field corresponding to `gather_perdb_stats' is not defined in `*mongodb.MongoDB'

I thought the latest release notes for Telegraf 1.0-Beta3 said this feature was enabled. Any ideas?